pyproject.toml

```
@@ -0,0 +1,11 @@
+[project]
+name = "resume"
+version = "0.1.0"
+description = "John Does's resume as rendered w/ RenderCV"
+requires-python = ">=3.14"
```

Oops, should be:

```suggestion
requires-python = ">=3.12"
```

to match the change-as-little-as-possible I went for in devcontainer.json (or conversely update that to pull a 3.14 tag; I'd also go alpine probably) 

I believe also determines version in rendercv pipeline?